pkg/ui/src/views/shared/components/sortabletable/index.tsx

```diff
@@ -128,16 +155,49 @@ export class SortableTable extends React.Component<TableProps, {}> {
             "sort-table__row",
             "sort-table__row--body",
             this.props.rowClass(rowIndex),
+            { "sort-table__row--expandable": !!expandableConfig },
           );
-          return (
```

Comment:
This per row function is getting a little unwieldy inline here in render.  Maybe worth pulling it out.

Reply:
Done